server/service/src/log.rs

```diff
@@ -21,3 +26,19 @@ pub fn get_logs(
         count: i64_to_u32(repository.count(filter)?),
     })
 }
+
+pub fn log_invoice_created(
+    connection: &StorageConnection,
+    user_id: String,
+    record_id: String,
+) -> Result<(), RepositoryError> {
```

Comment:
sorry didn't meant this with helper method thought more about:

`LogRowRepository::new(&connection).upsert_one(&LogRow{...})` -> `log(&connection, &LogRow {})`

(which only safes a little bit of typing though...)